apps/studio/src/server/modules/resource/resource.router.ts

```diff
@@ -230,13 +230,15 @@ export const resourceRouter = router({
           .where("Resource.type", "!=", ResourceType.RootPage)
           .where("Resource.type", "!=", ResourceType.FolderMeta)
           .where("Resource.type", "!=", ResourceType.CollectionMeta)
+          .where("Resource.type", "!=", ResourceType.CollectionLink)
           .where("Resource.siteId", "=", Number(siteId))
           .$narrowType<{
             type: Exclude<
               ResourceType,
               | typeof ResourceType.RootPage
               | typeof ResourceType.FolderMeta
               | typeof ResourceType.CollectionMeta
+              | typeof ResourceType.CollectionLink
             >
```

Comment:
should we take this chance to instead whitelist `=` instead of blacklist `!=` what should be returned?

since adding things that they should not add is worse than not being able to find things they should add